python/baseline/services.py

```diff
@@ -122,7 +140,8 @@ def load(cls, bundle, **kwargs):
         name = kwargs.get("name", None)
         if remote:
             beam = kwargs.get('beam', 10)
-            model = Service._create_remote_model(directory, be, remote, name, cls.signature_name(), beam, preproc=kwargs.get('preproc', False))
+            model = Service._create_remote_model(directory, be, remote, name, cls.signature_name(), beam,
+                                                 preproc=kwargs.get('preproc', False))
```

Comment:
This seems like it should default to `client` like the `kwargs.get('preproc', 'client')` does

Reply:
right. resolved.